templates/smoot_usage_raw.sql.py

```diff
@@ -144,36 +149,70 @@ def sql(self):
 BASE_SELECT = {
     "clients_last_seen_v1": """\
     SELECT
-      * EXCEPT(channel, normalized_channel),
-      normalized_channel AS channel
+      * REPLACE(normalized_channel AS channel)
     FROM
       clients_last_seen_v1""",
     "core_clients_last_seen_v1": """\
     SELECT
-      * EXCEPT(normalized_channel, metadata_app_version),
-      normalized_channel AS channel,
-      metadata_app_version AS app_version
-    FROM
-      core_clients_last_seen_v1
+      *,
+      normalized_channel AS channel
+    FROM (
```

Comment:
nested queries of this type [go against the sql style guide](https://docs.telemetry.mozilla.org/concepts/sql_style.html#nested-queries). We should either make this union a view and query that, or modify this script to create a CTE for this (e.g. by handling a list of strings here)